packages/suite/src/views/settings/debug/OAuthApi.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import GoogleClient from '@suite/services/google';
+import { ActionColumn, ActionSelect, SectionItem, TextColumn } from '@suite-components/Settings';
+import * as suiteActions from '@suite-actions/suiteActions';
+import { useSelector, useActions } from '@suite-hooks';
+import type { OAuthServerEnvironment } from '@suite-types/metadata';
+import { useAnchor } from '@suite-hooks/useAnchor';
+import { SettingsAnchor } from '@suite-constants/anchors';
+
+const StyledActionSelect = styled(ActionSelect)`
+    min-width: 256px;
+`;
+
+export const OAuthApi = () => {
+    const { setDebugMode } = useActions({
+        setDebugMode: suiteActions.setDebugMode,
+    });
+    const { debug } = useSelector(state => ({
+        debug: state.suite.settings.debug,
+    }));
+    const { anchorRef, shouldHighlight } = useAnchor(SettingsAnchor.InvityApi);
+
+    const options = Object.entries(GoogleClient.servers).map(([environment, server]) => ({
+        label: server,
+        value: environment,
+    }));
+    const selectedOption =
+        options.find(option => option.value === debug.oauthServerEnvironment) || options[0];
+
+    const handleChange = (item: { value: OAuthServerEnvironment }) => {
+        setDebugMode({
+            oauthServerEnvironment: item.value,
+        });
+        GoogleClient.setEnvironment(item.value);
+    };
+
+    return (
+        <SectionItem
+            data-test="@settings/debug/oauth-api"
+            ref={anchorRef}
+            shouldHighlight={shouldHighlight}
+        >
+            <TextColumn
+                title="API server"
+                description="Set the server url for generating Google refresh token"
```

Comment:
Would be nice to mention "labeling" here
```suggestion
                title="Google Auth server"
                description="Set the authorization server for labeling in Google Drive"
```

Reply:
Done in 469efbf93ea70aef646477c7b77bf76e1481be54